package.json

```diff
@@ -154,19 +156,12 @@
       "scripts/package.json"
     ],
     "versionGroups": [
-      {
```

Comment:
Not directly related but this is no longer needed; as of some previous PR test-bundles now has the same webpack version as everything else